src/contexts/auth-context.js

```diff
@@ -0,0 +1,67 @@
+import { createContext, useContext, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { signInService, signupService } from '../services';
+
+const AuthContext = createContext();
+
+const AuthProvider = ({ children }) => {
+    const navigate = useNavigate();
+
+    const getInitalAuthState = () => (localStorage.getItem('AUTH_TOKEN') ? true : false);
+
+    // Hanlder for signing in.
+    const signInHandler = async (event, userInput) => {
+        event.preventDefault();
+
+        const { email, password } = userInput;
+
+        try {
+            const responseToken = await signInService(email, password);
+            localStorage.setItem('AUTH_TOKEN', responseToken);
+            setAuth(true);
+            navigate('/');
+        } catch (error) {
+            console.log("Couldn't get token: ", error);
+        }
+    };
+
+    // Hanlder for signing up.
+    const signUpHandler = async (event, userInput) => {
+        event.preventDefault();
+
+        const {firstName, lastName, email, password} = userInput; 
+        console.log("created acc with: ", firstName, email, password)
+        try{
```

Comment:
remove the consoles in case u forgot to do so

Reply:
Those did get left out. Thanks for pointing it out!